scss/_patterns_footer.scss

```diff
@@ -4,11 +4,11 @@
   .p-footer {
     border-top: 1px dotted $color-x-dark;
     font-size: .75rem;
+    margin: 0 0 1rem;
```

Comment:
Do we want to remove margin that does not exist at this level? This could cause issues setting margins locally on the site. We should be prescriptive with are rules. Maybe we just need `margin-bottom: 1rem;`.